idaes/core/ui/fsvis/static/js/stream_table.js

```diff
@@ -87,13 +170,40 @@ export class StreamTable {
                         row_object[variable_col] = row_object[variable_col] + '<span class="streamtable-units">&ndash;</span>';
                     }
                 }
+                else if (columns[col_index].includes("_vartype")) {
```

Comment:
ok, so you're decoding your own encoding of the column name -- again, wouldn't this be easier by just passing an extra list of types that matches the columns; or alternatively, a dictionary of `{column_name: type}` ?